home/javascript/react/components/FigureProteinSummary.js

```diff
@@ -5,7 +5,7 @@ import {publicationType} from '../utils/types';
 import PublicationCitationLink from './PublicationCitationLink';
 import CommaSeparatedList from './CommaSeparatedList';
 
-const FigureSummary = ({statistics, fishID}) => {
+const FigureSummary = ({statistics, fishID, markerID =('')}) => {
```

Comment:
I don't think the parentheses do anything here: markerID =('') ?

Reply:
Just setting the markerID to an empty string if not set at initialization time (opposed to a null).